guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,600 @@
+| AKKA_REMOTING_BIND_PORT | The Akka remoting port.             | 2551          |
+| AKKA_SEED_NODE_HOST     | The hostname of the first container in the StatefulSet. | `myapp-0` |
+| AKKA_SEED_NODE_PORT     | The Akka remoting port of the seed node. In most cases this value should match `AKKA_REMOTING_BIND_PORT`. | 2551 |
+| AKKA_ACTOR_SYSTEM_NAME  | The name of the `ActorSystem` of your application. For the purpose of this guide, we'll match the `ActorSystem` name with the application name. | 'myapp' |
+
+
+The following code can be used by your application to set up the `ActorSystem` based on the name specified by `actorSystemName` system property:
+
+```
+val actorSystemName = sys.props.get("actorSystemName")
+  .getOrElse(throw new IllegalArgumentException("Actor system name must be defined"))
+
+val actorSystem = ActorSystem(actorSystemName)
+```
+
+
+## 3. Publishing to Docker registry
+
+Next we will containerize the application and publish its Docker image. Please proceed with either [SBT instructions](#Publishing-to-Docker-registry---SBT) or [Maven instructions](#Publishing-to-Docker-registry---Maven) accordingly.
+
+
+### 3.1 Publishing to Docker registry - SBT
+
+We will be using the [SBT Native Packager](http://www.scala-sbt.org/sbt-native-packager/) plugin to containerize the application.
+
+Enable SBT Native Packager in your project by adding the following line in the `project/plugins.sbt`:
+
+```
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.2.0")
+```
+
+Next, follow the steps for a [SBT multi-module project](SBT-multi-module-project) or [SBT single-module project](SBT-single-module-project).
+
+
+#### 3.1.1 SBT multi-module project
+
+Follow this section if your application is part of a multi-module SBT project.
+
+Enable the [JavaAppPackaging](http://www.scala-sbt.org/sbt-native-packager/archetypes/java_app/index.html) plugin provided by SBT Native Packager by adding the `enablePlugins` instruction to the module declaration in the `build.sbt`:
+
+```
+lazy val myApp = project("my-app")
+  // Add the following line to the module definition
+  .enablePlugins(JavaAppPackaging)
+```
+
+Alternatively you may choose to enable [JavaServerAppPackaging](http://www.scala-sbt.org/sbt-native-packager/archetypes/java_server/index.html) instead:
+
+```
+lazy val myApp = project("my-app")
+  // Add the following line to the module definition
+  .enablePlugins(JavaServerAppPackaging)
+```
+
+The `JavaServerAppPackaging` plugin provides all the features provided by the `JavaAppPackaging` plugin with some additional [server features](http://www.scala-sbt.org/sbt-native-packager/archetypes/java_server/index.html#features) such as daemon user/group support and support for `/etc/default`.
+
+Please note that you must choose to enable either `JavaAppPackaging` or `JavaServerAppPackaging` - you _can't_ enable both.
+
+We will now configure the Docker settings required to containerize the application.
```

Comment:
Replace "Append ..." with:
2. Add a `dockerEntrypoint` to module settings that supplies the system properties necessary to containerize the application: